src/sweeper/sweepers/UseLimitations.html

```diff
@@ -1,10 +1,10 @@
 <p>
-    The data herein, including but not limited to geographic data, tabular data, analytical data, electronic data structures or files, are provided &quot;as is&quot; without warranty of any kind, either expressed or implied, or statutory, including, but not limited to, the implied warranties or merchantability and fitness for a particular purpose. The entire risk as to the quality and performance of the data is assumed by the user.
+    The data, including but not limited to geographic data, tabular data, and analytical data, are provided &quot;as is&quot; and &quot;as available&quot;, with no guarantees relating to the availability, completeness, or accuracy of data, and without any express or implied warranties. These data are provided as a public service for informational purposes only. You are solely responsible for obtaining the proper evaluation of a location and associated data by a qualified professional. AGRC reserves the right to change, revise, suspend or discontinue published data and services without notice at any time. Neither AGRC nor the State of Utah are responsible for any misuse or misrepresentation of the data. AGRC and the State of Utah are not obligated to provide you with any maintenance or support. The user assumes the entire risk as to the quality and performance of the data. You agree to hold the State of Utah harmless for any claims, liability, costs, and damages relating to your use of the data. You agree that your sole remedy for any dissatisfaction or claims is to discontinue use of the data.
 </p>
 
 <p>
-    <a href="http://creativecommons.org/licenses/by/4.0/">
-        <img alt="Creative Commons License" src="https://i.creativecommons.org/l/by/4.0/88x31.png" style="border-width:0;" />
+    <a href="http://creativecommons.org/licenses/by/4.0/" rel="nofollow and ugc">
```

Comment:
**issue**: `and` is not used to add multiple relationships. I'm starting to think that esri modified the content when uploading. We will have to check to make sure we don't foul up any checks we may do in the future but I think the following is more correct. 
```suggestion
    <a href="http://creativecommons.org/licenses/by/4.0/" rel="license nofollow noopener ugc">
```